lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -137,6 +137,13 @@ public function get_items( $request ) {
 			$prepared_args['offset'] = $prepared_args['number'] * ( absint( $request['page'] ) - 1 );
 		}
 
+		$allowed_filters = array( 'post_name', 'post_type', 'post_author', 'post_parent', 'post_status', 'type__in', 'type__not_in', 'date_query' );
```

Comment:
I think we need these filters behind `edit_posts`, or even generally think about capabilities more, because a public request shouldn't be able to query for the comments of a private post.